templates/deno/app/entry.client.tsx

```diff
@@ -1,3 +1,6 @@
+/// <reference no-default-lib="true" />
+/// <reference lib="dom" />
```

Comment:
I think this should be in a `tsconfig.json` instead? 🤔

CC/ @pcattori

Reply:
Deno doesn't use `tsconfig.json`, but we could instead include this in a [`deno.json` config file](https://deno.land/manual/getting_started/configuration_file).